### DISCUSSION TITLE
¿Alguien sabe qué pasó con las katas 7-10?

### DISCUSSION THREAD

**Discussion by noelsilis** [LaunchX-InnovaccionVirtual/CursoIntroPython#98] [Q&A] [answered]
Buenas tardes a todos. De casualidad alguien sabe que sucedio con las Katas que se habilitarían hoy?.

**Comment by GabrielVillagran** [comment, marked as answer]
Se habilitarán hasta mañana, no recuerdo si las 2 o solo 1, pero esto se hará porque hay personas que aún ingresarán con nosotros y poder ir todos iguales

**Reply by noelsilis** [reply]
Ok, muchas gracias

**Comment by RodolfoMorquecho** [comment]
Dijo la profesora que las liberaba hoy por la noche, saludos.

**Reply by noelsilis** [reply]
OK. Muchas gracias

**Reply by 3rickDJ** [reply]
Hi @noelsilis  :safety_pin: , podrías marcar tu pregunta como respondida, para aquellos que necesiten que su pregunta sea respondida :smile:

**Comment by FernandaOchoa** [comment]
> Dime que no entras a las sesiones en vivo, ni ves las repeticiones. Sin decirme que no entras a las sesiones en vivo, ni ves las repeticiones.
